.github/workflows/docker.yml

```diff
@@ -1,16 +1,12 @@
 name: Docker Images
 
+# This workflow needs permissions to publish images to ghcr.io,
+# so it does not work for forks. Therefore, we only trigger it
+# on pushes to aiidateam/aiida-core repo.
 on:
-  pull_request:
-    paths-ignore:
-    - '**.md'
-    - '**.txt'
-    - docs/**
-    - tests/**
```

Comment:
Why remove this though? Surely if we just change docs there is no reason to trigger the build, is there?

Reply:
Huh? I removed the whole pull_request event. :-) The same filters are applied on push event just below.

btw: Feel free to pull these changes into #6445

Reply:
😅 never mind, they were nicely hidden by Github's feature to only show a minimal subset of the file affected

Reply:
Yes, I also love that feature!